charts/manticoresearch/templates/manticore-balancer.yaml

```diff
@@ -92,6 +92,13 @@ spec:
           - name: config-volume
             mountPath: {{ .Values.balancer.config.path }}
             subPath: manticore.conf
+          {{- range .Values.balancer.volumeMounts }}
+          - name: {{ .name }}
+            mountPath: {{ .mountPath }}
```

Comment:
Can you add prefix dir to these mounts? To prevent volume mapping `/var/lib/manticore` or other important directories.

Like this `mountPath: /imports/{{ .mountPath }}`

Reply:
Done!